keras/ops/core_test.py

```diff
@@ -505,3 +506,249 @@ def test_convert_to_tensor(self, x, dtype, expected_dtype):
                 ),
                 expected_dtype,
             )
+
+
+class CoreOpsCallsTests(testing.TestCase):
```

Comment:
What is the purpose of the new tests?

Reply:
I usually use Codecov to identify untested lines and then write tests for them. Afterward, I check the report again to see the changes reflected.

https://app.codecov.io/gh/keras-team/keras/pull/19119/blob/keras/ops/core.py